ci/run_tests/run_tests_windows.py

```diff
@@ -1,14 +1,26 @@
 import os
 import argparse
 import subprocess
-import netifaces as ni
+import scapy.arch.windows
 
 TCPREPLAY_PATH = "tcpreplay-4.4.1-win"
 PCAP_FILE_PATH = os.path.abspath(
     os.path.join("Tests", "Pcap++Test", "PcapExamples", "example.pcap")
 )
 
 
+def get_ip_by_guid(guid):
+    interfaces = scapy.arch.windows.get_windows_if_list()
+    for iface in interfaces:
+        if iface["guid"] == guid:
+            if len(iface["ips"]) > 0:
+                # iface["ips"] will put ipv6 address first
+                return iface["ips"][1]
```

Comment:
1. Since we're taking the second element from `iface["ips"]`, don't we want to check `if len(iface["ips"]) > 1`?
2. A nicer way to do it can be:
   ```python
   return iface["ips"][1] if len(iface["ips"]) > 1 else None
   ```

Reply:
Sure. I will change it.